SUI/SUI/Verification/VerificationViewModel.swift

```diff
@@ -0,0 +1,79 @@
+// VerificationViewModel.swift
+// Copyright © RoadMap. All rights reserved.
+
+import SwiftUI
+
+/// Вью модель экрана верификации
+final class VerificationViewModel: ObservableObject {
+    // MARK: - Private Constants
+
+    private enum Constants {
+        static let emptyString = ""
+        static let startValueNumber: Float = 0
+        static let maxValueNumber: Float = 10
+        static let timeIntervalNumber: Float = 1
+        static let oneNumber = 1
+    }
+
+    // MARK: - Public properties
+
+    @Published var isHidden = true
+    @Published var value: Float = Constants.startValueNumber
+    @Published var isContinueVerification = false
+    @Published var isAlertShowCode = false
```

Comment:
```suggestion
    @Published var isAlertCodeShown = false
```
Саша и Женя писали что обязательно должно начинаться на is и заканчиваться на shown